testrunner/execute.go

```diff
@@ -117,15 +117,18 @@ func getStructure(lines bytes.Buffer, input_dir string, ver int, taskIDsEnabled
 	return report
 }
 
-func buildTests(lines bytes.Buffer, input_dir string) ([]testResult, error) {
+func processTestResults(lines bytes.Buffer, input_dir string) ([]testResult, error) {
```

Comment:
Renamed because the old name sounded like something would be build via "go build" which is not the case.